docs/tutorials/t1.rst

```diff
@@ -102,6 +108,62 @@ using ``child_data``
         for result in sub_data.analysis_results():
             print(result)
 
+:math:`T_1` experiments with kerneled measurement
+---------------------------------------------------
+:math:`T_1` experiment can also be done with kerneled measurement.
+If we will set the running option `meas_level=MeasLevel.KERNELED`, the
+job will not discriminate the data and will not label it. In T1 experiment,
+as we know that in :math:`P(1|t=0)=1`, we will add measurement for delay=0
+and for extreme delay. In this configuration we know that the data starts from
+a point [I,Q] that is close to a logical value '1' and ends at a point [I,Q]
+that is close to a logical value '0'. Beside that, We will need to use analysis
+option to normalize the output.
+
+
+.. jupyter-execute::
+
+    # Experiment
+    ns = 1e-9
+    mu = 1e-6
+
+    # qubit properties
+    t1 = [45 * mu, 45 * mu]
+    t2 = [value/2 for value in t1]
+
+    # we will guess that our guess is 10% off the exact value of t1 for qubit 0.
+    t1_estimated_shift = t1[0]/10
+
+    # We use log space for the delays because of the noise properties
+    delays = np.logspace(1, 11, num=20, base=np.exp(1))
+    delays = [delay * ns for delay in delays]
+
+    # adding to the circuit 0 delay and long delays to help the curve fitting
```

Comment:
1. You're not adding these delays to "the circuit"; rather you're adding circuits with these delays.
2. You're not doing it in order to help the curve fitting (the curve fitting is doing fine without it), but because otherwise the fitting will provide wrong results (because the centers will be misplaced).

Reply:
Does this capture the essence of it?

```
# Adding circuits with delay=0 and long delays so the centers in the IQ plane won't be misplaced.  
# Without this, the fitting can provide wrong results.
```